lib/modules/manager/sbt/index.ts

```diff
@@ -3,8 +3,11 @@ import { SbtPackageDatasource } from '../../datasource/sbt-package';
 import { SbtPluginDatasource } from '../../datasource/sbt-plugin';
 import * as ivyVersioning from '../../versioning/ivy';
 
-export { extractPackageFile } from './extract';
-export { bumpPackageVersion } from './update';
+// extractPackageFile is used only for test and within extractAllPackageFiles FN
+// extractAllPackageFiles has more higher priority
+// lib/workers/repository/extract/manager-files.ts:32:0
+export { extractPackageFile, extractAllPackageFiles } from './extract';
```

Comment:
```suggestion
export { extractAllPackageFiles } from './extract';
```

Reply:
@zharinov `extractPackageFile`  is used by auto replace to verify right replacement

Reply:
I written `updateDependency` function in update.ts so it can update more accurate with `fileReplacePosition`

Reply:
then @zharinov is right, it should removed from public api